src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -210,7 +210,7 @@ def regenerate_keys(cmd, client, resource_group, name, type):
 
 
 def app_create(cmd, client, resource_group, service, name,
-               is_public=None,
+               assign_endpoint=None,
```

Comment:
what's the behavior if user passing `is_public` after the PR? no effect right? it should keep same before deprecation date.

Reply:
No effect right now.